narwhal/consensus/src/consensus.rs

```diff
@@ -31,10 +32,10 @@ pub type Dag = BTreeMap<Round, HashMap<PublicKey, (CertificateDigest, Certificat
 
 /// The state that needs to be persisted for crash-recovery.
 pub struct ConsensusState {
-    /// The last / highest committed round.
-    pub last_committed_round: Round,
-    /// The current GC round.
-    pub gc_round: Round,
+    /// The information about the last committed round and corresponding GC round.
+    pub last_round: CommittedRound,
```

Comment:
TBH the `last_round` and `CommittedRound` names are a bit confusing to me. How about
```
struct ConsensusRound {
    pub last_committed_round: Round,
    pub gc_round: Round,
}
```
Open to other suggestions too.

Reply:
Regarding the `last_round` , would it be better if that would be named `current_round` ? 

I chose `CommittedRound` to make it quite specific that the struct contains info about a round that we have performed a commit on. That's why I also avoided to include into the underlying names the `last_` . Let me see if I can come up with something better here. The `ConsensusRound` is good but it seems quite generic in terms of meaning for me - but honestly I am not strongly opinionated.

Reply:
I think last_round / current_round have other meanings in other parts of the codebase. For the commit action, maybe it does not have to be part of the name? But I'm also ok with
```
struct LastCommitted {
    pub round: Round,
    pub gc_round: Round,
}
```
Another alternative,
```
struct ConsensusRound {
    pub committed: Round,
    pub gc: Round,
}
```
The advantage of using `ConsensusRound` is that it matches the name of the channel it is passed on, which is nicer.